src/index.ts

```diff
@@ -42,6 +42,8 @@ export class Client {
   endpoint: string;
   key?: string;
   secret?: string;
+  // To allow override in React SDK
```

Comment:
`// Callers can set this to add an additional user agent value to the request's metadata.`